extra-libs/qt-6/autobuild/defines

```diff
@@ -0,0 +1,62 @@
+PKGNAME=qt-6
+PKGSEC=x11
+PKGDES="Qt version 6"
+PKGDEP="alsa-lib bluez cups dbus desktop-file-utils double-conversion \
+        fontconfig geoclue gst-plugins-base-1-0 harfbuzz hicolor-icon-theme \
+        hyphen icu libinput libjpeg-turbo libmng libpng libproxy libxkbcommon \
+        libxslt mesa nodejs openal-soft pcre2 protobuf pulseaudio re2 snappy \
+        sqlite srtp systemd tslib vulkan xdg-utils xcb-util-renderutil \
+        xcb-util-cursor xcb-util-wm xcb-util-image xcb-util-keysyms zlib"
+BUILDDEP="gtk-3 mariadb openssl postgresql unixodbc gperf ninja cmake"
+
+ABTYPE=cmakeninja
+# To query possible build options, you may want to check following files in
+# Qt 6 source tree: qtbase/configure, qtbase/config_help.txt,
+# qtbase/cmake/configure-cmake-mapping.md and
+# qtbase/cmake/QtProcessConfigureArgs.cmake.
+# It's also helpful to refer to build options used by other distributions.
+CMAKE_AFTER="-DINSTALL_BINDIR=bin \
+             -DINSTALL_DOCDIR=share/doc/qt6 \
```

Comment:
Should be `share/doc/qt-6`. This was a mistake we made in Qt 5, and wouldn't be easily fixed unless we run a survey for all the packages containing `/usr/share/doc/qt5`.